rules-tests/Symfony73/Rector/Class_/InvokableCommandInputAttributeRector/Fixture/command_with_io.php.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Rector\Tests\Symfony\Symfony73\Rector\Class_\InvokableCommandInputAttributeRector\Fixture;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand('app:my-command')]
+final class MyCommand extends Command
+{
+    #[\Override]
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $io->info('Great success!');
+
+        return 1;
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\Tests\Symfony\Symfony73\Rector\Class_\InvokableCommandInputAttributeRector\Fixture;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+ 
+#[AsCommand('app:my-command')]
+final class MyCommand
+{
+    #[\Override]
+    public function __invoke(\Symfony\Component\Console\Style\SymfonyStyle $io): int
```

Comment:
I think it should not be replaced, skip the change should be the way, right? /cc @darthf1 @thePanz @gharlan could you verify? Thank you.

Reply:
as this is a bugfix, I would keep a KISS solution: @samsonasik 's suggestion to not rewrite the parameters in the constructor looks reasonable as a first step.